modules/arm_plugin/dockerfiles/Dockerfile.RPi64_bionic

```diff
@@ -53,6 +53,7 @@ RUN apt-get update && \
         libavcodec-dev:arm64 \
         libavformat-dev:arm64 \
         libswscale-dev:arm64 \
+        patchelf \
```

Comment:
can we also update other docker files https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/arm_plugin/dockerfiles ? I suppose the change is similar

Reply:
done